docs/resources/cluster.md

```diff
@@ -10,8 +10,22 @@ description: |-
 
 Manages a cluster attached to an Argo CD instance.
 
-## Example Usage
+## Example Usage (Basic)
+```terraform
+resource "akp_instance" "argocd" {
+  name = "argocd"
+  argocd = {
+    "spec" = {
+      "instance_spec" = {
+        "declarative_management_enabled" = true
+      }
+      "version" = "v2.8.3-ak.2"
+    }
+  }
+}
+```
```

Comment:
I guess this example(for instance) should be added to the `docs/resources/instance.md` file while the next example(for cluster) should be added here?

Reply:
Oh yeah, I mixed that up! I'll fix it